### PR TITLE
Autopsy scanner & optable sound

### PR DESCRIPTION
**Pull request by alsoandanswer**
also tidies up some sounds here and there to use the proper list in sounds.dm instead of single files.

and again, kills more [span class].

autopsy scanner looks more medical now.

### CODE REVIEW

**Comment by alsoandanswer** [comment]
!review

File: code/game/objects/items/weapons/autopsy.dm [hunk 1]
```diff
@@ -173,19 +174,19 @@
 		src.wdata = list()
 		src.chemtraces = list()
 		src.timeofdeath = null
-		to_chat(user, "<span class='notice'>A new patient has been registered.. Purging data for previous patient.</span>")
+		to_chat(user, span("notice", "A new patient has been registered.. Purging data for previous patient."))
```
Comment (Arrow768) [inline code/game/objects/items/weapons/autopsy.dm]:
```suggestion
		to_chat(user, span("notice", "A new patient has been registered. Purging data for previous patient."))
```

File: code/game/machinery/OpTable.dm [hunk 2]
```diff
@@ -52,24 +52,25 @@
 
 /obj/machinery/optable/attack_hand(mob/user as mob)
 	if (HULK in usr.mutations)
-		visible_message("<span class='danger'>\The [usr] destroys \the [src]!</span>")
+		visible_message(span("danger", "\The [usr] destroys \the [src]!"))
```
Comment (NonQueueingMatt) [inline code/game/machinery/OpTable.dm]:
```suggestion
		visible_message(span("danger", "\The [user] destroys \the [src]!"))
```

File: code/game/machinery/OpTable.dm [hunk 3]
```diff
@@ -52,24 +52,25 @@
 
 /obj/machinery/optable/attack_hand(mob/user as mob)
 	if (HULK in usr.mutations)
```
Comment (NonQueueingMatt) [inline code/game/machinery/OpTable.dm]:
```suggestion
	if (HULK in user.mutations)
```

File: code/game/machinery/OpTable.dm [hunk 4]
```diff
@@ -52,24 +52,25 @@
 
 /obj/machinery/optable/attack_hand(mob/user as mob)
```
Comment (NonQueueingMatt) [inline code/game/machinery/OpTable.dm]:
```suggestion
/obj/machinery/optable/attack_hand(mob/user)
```

**Pull request merged**